packages/form-js-editor/test/spec/properties-panel/groups/GeneralGroup.spec.js

```diff
@@ -367,21 +367,115 @@ describe('GeneralGroup', function() {
     });
 
 
-    it('should render for INPUTS', function() {
+    describe('for singleSelect-like INPUTS', () => {
 
-      // given
-      for (const type of INPUTS.filter(i => ![ 'checklist', 'taglist' ].includes(i))) {
+      const singleSelectInputTypes = ['radio', 'select'];
```

Comment:
```suggestion
      const singleSelectInputTypes = [ 'radio', 'select' ];
```

Mh, don't we lint that?

Reply:
Damn, we still use `0.12` of our eslint rules, so the [bracket rules](https://github.com/bpmn-io/eslint-plugin-bpmn-io/commit/9dba9c9b29d28ed47ea7a471bd252e59231066d3) don't apply yet.